aap-common/proc-aap-pull-command-container-image.adoc

```diff
@@ -4,12 +4,22 @@
 
 Pull the Docker image for the Ansible on Clouds operational container with the same tag version as your deployment.
 
-For example, if your foundation deployment version is 2.2.20230215, you must pull the operational image with tag 2.2.20230221-00.
+[NOTE]
+====
+Before Pulling the docker image, make sure you are logged in to redhat registry using docker. Use the following command to login to registry.redhat.com.
+[source,bash]
+----
+$ docker login registry.redhat.io
+----
+For more information about registry login, see link:https://access.redhat.com/RegistryAuthentication[Registry Authentication]
+====
+
+For example, if your foundation deployment version is 2.3.20230221-00, you must pull the operational image with tag 2.3.20230221.
```

Comment:
Should there be a -00 at the end?

Reply:
The tag used to pull will not have -00 (due to downstream build publication limitations in the tooling)